web-gateway/app/controllers/CreateUserForm.java

```diff
@@ -14,4 +26,12 @@ public String getName() {
     public void setName(String name) {
         this.name = name;
     }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
```

Comment:
Most user registration forms include duplicate password and "Verify Password" fields that have to match, to make sure they don't enter the password with a typo and then are unable to log in.